### PR TITLE
Fix sample_ctl.py

### PR DESCRIPTION
**Pull request by nneonneo**
`sample_ctl.py` was not running correctly. It was crashing with various errors: `TypeError: Uc.ctl_remove_cache() missing 1 required positional argument: 'end'`, `KeyError: 64`. This patch fixes the relevant errors in the bindings and scripts.

### CODE REVIEW

**Comment by wtdcode** [comment]
Hi, thanks for your interest! Note the patch should go to dev branch.

**Comment by nneonneo** [comment]
@wtdcode fixed now.

File: bindings/python/unicorn/unicorn.py [hunk 1]
```diff
@@ -424,6 +424,9 @@ class uc_arm64_cp_reg(ctypes.Structure):
         ("val", ctypes.c_uint64)
     ]
 
+    def __repr__(self):
```
Comment (wtdcode) [inline bindings/python/unicorn/unicorn.py:427]: If you would like to add `__repr__` for `uc_arm64_cp_reg`, could you do the same for `uc_arm_cp_reg`?

**Comment by nneonneo** [comment]
@wtdcode ok, I just added `__repr__` to all of the Structure subclasses, hope that's useful.

**Comment by wtdcode** [comment]
LGTM, thanks!

**Pull request merged**